src/reflection/skrypnyk/Student.java

```diff
@@ -0,0 +1,58 @@
+package src.reflection.skrypnyk;
+
+public class Student {
```

Comment:
Firstly, fields of class should be placed in first order, after that public methods(public API of class) and after public methods -- private methods.
After equals hashode/toString methods.